tests/contrib/test_xsrfutil.py

```diff
@@ -289,6 +290,17 @@ def testGenerateAndValidateToken(self):
                                                  TEST_USER_ID_1,
                                                  action_id=TEST_ACTION_ID_1))
 
+        # When the token is a string
+        if six.PY2:
+            str_token = str(token)
+        elif six.PY3:
+            str_token = token.decode('utf-8')
```

Comment:
In py2 return a unicode string (u'string') and not a native string. In py3 return a native string because native strings are unicode strings.